examples/onnxrt/object_detection/onnx_model_zoo/tiny_yolov3/quantization/ptq/main.py

```diff
@@ -159,27 +339,63 @@ def __call__(self, sample):
+        session = ort.InferenceSession(model.SerializeToString(), None)
+        ort_inputs = {}
+        len_inputs = len(session.get_inputs())
+        inputs_names = [session.get_inputs()[i].name for i in range(len_inputs)]
+        for idx, (inputs, labels) in enumerate(dataloader):
+                if not isinstance(labels, list):
+                    labels = [labels]
+                if len_inputs == 1:
+                    ort_inputs.update(
+                        inputs if isinstance(inputs, dict) else {inputs_names[0]: inputs}
+                    )
+                else:
+                    assert len_inputs == len(inputs), 'number of input tensors must align with graph inputs'
+                    if isinstance(inputs, dict):
+                        ort_inputs.update(inputs)
+                    else:
+                        for i in range(len_inputs):
+                            if not isinstance(inputs[i], np.ndarray):
+                                ort_inputs.update({inputs_names[i]: np.array(inputs[i])})
+                            else:
+                                ort_inputs.update({inputs_names[i]: inputs[i]})
+                predictions = session.run(None, ort_inputs)
+                predictions, labels = postprocess((predictions, labels))
+                metric.update(predictions, labels)
+        return metric.result()
+
     if args.benchmark:
-        from neural_compressor.experimental import Benchmark, common
-        evaluator = Benchmark(args.config)
-        evaluator.model = common.Model(model)
-        evaluator.b_dataloader = dataloader
-        evaluator.postprocess = common.Postprocess(Post)
-        evaluator(args.mode)
+        if args.mode == 'performance':
+            from neural_compressor.benchmark import fit
+            from neural_compressor.config import BenchmarkConfig
+            conf = BenchmarkConfig(iteration=100,
+                                cores_per_instance=4,
+                                num_of_instance=1)
+            fit(model, conf, b_dataloader=dataloader)
+        elif args.mode == 'accuracy':
+            acc_result = eval_func(model)
+            print("Batch size = %d" % args.batch_size)
+            print("Accuracy: %.5f" % acc_result)
+
 
     if args.tune:
-        from neural_compressor import options
-        from neural_compressor.experimental import Quantization, common
-        options.onnxrt.graph_optimization.level = 'ENABLE_BASIC'
-
-        quantize = Quantization(args.config)
-        quantize.model = common.Model(model)
-        quantize.eval_dataloader = dataloader
-        quantize.calib_dataloader = dataloader
-        quantize.postprocess = common.Postprocess(Post)
-        q_model = quantize()
+        from neural_compressor import quantization, PostTrainingQuantConfig
+        from neural_compressor.config import AccuracyCriterion
+        accuracy_criterion = AccuracyCriterion()
+        accuracy_criterion.criterion =' absolute'
```

Comment:
![image](https://user-images.githubusercontent.com/51692656/208103214-30d77a1f-3bba-47ba-8a45-48254a35c58f.png)
please remove the space.